packages/wdio-browserstack-service/src/util.ts

```diff
@@ -1294,18 +1294,28 @@ export const patchConsoleLogs = o11yErrorHandler(() => {
     const BSTestOpsPatcher = new logPatcher({})
 
     Object.keys(consoleHolder).forEach((method: keyof typeof console) => {
+        if (!(method in console) || typeof console[method] !== 'function') {
```

Comment:
Adding the check here
```suggestion
        if (!(method in console) || method === 'Console' || typeof console[method] !== 'function') {
```